template/lib.typ

```diff
@@ -136,19 +135,28 @@
   )
 }
 
-#let index() = context {
-    [= Índice]
+
+#let index(title: [Índice], target: heading, lvl: 1) = context {
```

Comment:
I think that the usage of `lvl` here is superfluous. If you want to match headings of level 1 or 2 for e.g. an abbreviated index you can do `heading.where(level: 1).or(heading.where(level: 2))` as the selector.

Reply:
Oh I didnt know you could do that. I didnt found it in the documentation